src/release-info/intro.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<preambles>
+  <eac>
+  <!--
+ 
+                              An XML Schema:
+  Encoded Archival Context - Corporate Bodies, Persons, and Families (EAC-CPF)
+
+  See: https://eac.staatsbibliothek-berlin.de/schemata-and-tag-library/
+
+  *****************************************************************
+  *                                                               *                
+  *                            EAC-CPF 2.0                        *
+  *                                                               *
+  *                           2022 July 17                        *          
```

Comment:
Not sure about the date, should it be 3rd of August or the day council approved (May 18th)?

Reply:
I'm leaning toward just leaving it out altogether.  Let's discuss tomorrow.